spec/dataframe_spec.rb

```diff
@@ -3029,6 +3029,17 @@
     end
   end
 
+  context "#reset_index!" do
+    it "resets index array of dataframe" do
+      data = {x: [5,1,-6], y: [2,7,0], z: [4,3,8]}
+      idx = ["u","v","w"]
+      df1 = Daru::DataFrame.new(data, index: idx)     
+      df2 = Daru::DataFrame.new(data, index: Daru::Index.new(3))     
+      df1.reset_index!
+      expect(df1.index.to_a).to eq(df2.index.to_a)
+    end
+  end
```

Comment:
I understand you are looking at other tests around, but we are trying to write proper tests for new functionality, with all modern RSpec approaches: http://betterspecs.org/ (E.g. `let`/`subject` and so on)